Packs/Intezer/Integrations/IntezerV2/IntezerV2.py

```diff
@@ -506,10 +507,8 @@ def enrich_dbot_and_display_url_analysis_results(intezer_result, intezer_api):
         intezer_result['redirect_chain'] = redirect_chain
 
     if 'indicators' in intezer_result:
-        indicators: Dict[str, List[str]] = {}
+        indicators: Dict[str, List[str]] = defaultdict(list)
```

Comment:
Why did you remove the fix you previously added here?

Reply:
After you comment, I just used a different solution to the issue which is default dict

Reply:
I think I prefer the previous solution because using the 'defaultdict' package entails a change to the py3-tools docker image and I prefer not to do so. This may delay the PR unnecessarily.
Your previous solution is good enough.

Reply:
`defaultdict` is part of `collections` package which is a built-in package in python so new packages in the docker image 🙂

Reply:
okay - good to know! :) 
I'll update you about the build issue on Sunday. Have a great weekend!